src/pages/login/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import axios from "axios";
+import Router from "next/router";
+import { Form, Input, Button, Checkbox } from "antd";
+import { UserOutlined, LockOutlined } from "@ant-design/icons";
+import Link from "next/Link";
+
+import style from "./login.module.css";
+
+interface Value {
+  email: string;
+  password: string;
+}
+
+const Login = () => {
+  const auth = async (value: Value) => {
```

Comment:
you dont need this function to be in the `Login` component, suggest to move it outside